src/Tribe/Tickets_Handler.php

```diff
@@ -1195,6 +1195,14 @@ public function get_event_rsvp_tickets( $post = null ) {
 	 *             (allows zero but not `-1` for Unlimited). If oversold, will be corrected to zero.
 	 */
 	public function get_ticket_max_purchase( $ticket_id ) {
+
+		$cache = tribe( 'cache' );
+		$key   = __METHOD__ .'-'. $ticket_id;
```

Comment:
* :no_entry_sign: Concat operator must be surrounded by a single space
* :no_entry_sign: Concat operator must be surrounded by a single space

<sub>hash: 5b227172fea519752362978e6712f7fe</sub>

Reply:
Lets make sure we solve this.

Reply:
```suggestion
		$key   = __METHOD__ . '-' . $ticket_id;
```